doc/rofi-theme.5.markdown

```diff
@@ -594,28 +593,28 @@ The following properties are currently supports:
 
 * **background**:       color
 * **foreground**:       color
-* **font**:             The font used by this textbox. (string)
-* **str**:              The string to display by this textbox. (string)
-* **vertical-align**:   Vertical alignment of the text. (0 top, 1 bottom)
-* **horizontal-align**: Horizontal alignment of the text. (0 left, 1 right)
-* **text**:             The text color to use (falls back to foreground if not set)
-* **highlight**:        Text Style {color}
-    Color is optional, multiple highlight styles can be added like:  bold underline italic #000000;
-* **width**:            Override the desired width for the textbox.
+* **font**:             the font used by this textbox (string)
+* **str**:              the string to display by this textbox (string)
+* **vertical-align**:   vertical alignment of the text (`0` top, `1` bottom)
+* **horizontal-align**: horizontal alignment of the text (`0` left, `1` right)
+* **text**:             the text color to use (falls back to foreground if not set)
+* **highlight**:        text style {color}
+    color is optional, multiple highlight styles can be added like: bold underline italic #000000;
+* **width**:            override the desired width for the textbox
 
 ### listview:
 * **columns**:         integer
-    Number of columns to show (atleast 1).
+    Number of columns to show (at least 1)
 * **fixed-height**:    boolean
-    Always show `lines`  rows, even if less elements are available.
+    Always show `lines` rows, even if fewer elements are available.
 * **dynamic**:         boolean
-    If the size should changed when filtering the list, or if it should keep the original height.
+    If the size should change when filtering the list, or if it should keep the original height.
```

Comment:
Correct.  If option is set (true) the size will change while filtering the list.